sites/.common/src/scss/common.scss

```diff
@@ -73,3 +73,12 @@ a {
 footer {
   padding: 1% 5%;
 }
+
+.hero.is-fullheight.is-not-fullheight {
+  min-height: 95vh;
+}
+
+.container.has-flex-align-items-center {
+  display: flex;
+  align-items: center;
+}
```

Comment:
This are the CSS rules that made the 404 page look centered on the landing site, but not on the tv one. I just moved them to `common.scss` and that made the trick.